compiler/one-cmds/one-build

```diff
@@ -154,25 +154,30 @@ def main():
     config = _parse_cfg(args)
 
     # verify configuration file
-    drivers = [
-        'one-import-tf', 'one-import-tflite', 'one-import-bcq', 'one-import-onnx',
-        'one-optimize', 'one-quantize', 'one-pack', 'one-codegen'
+    import_drivers_dict = _utils._detect_one_import_drivers()
+    other_drivers = [
```

Comment:
(optioanl) maybe `transform_drivers` ?

Reply:
Does this imply that `_detect_one_import_drivers` should be `_detect_transform_drivers` ?

Reply:
Um? I think `import_drivers_dict = _utils._detect_one_import_drivers()` is OK for import drivers. I think `transform_drivers` would be better for `other_drivers`. If you ment provide`_detect_transform_drivers` method for transform drivers, this is not what I thought.

Reply:
Uh, I messed it up...
Thank you for clarification =)